packages/frontend/src/components/common/Filters/FiltersProvider.tsx

```diff
@@ -69,9 +69,7 @@ export const FiltersProvider = <T extends DefaultFieldsMap = DefaultFieldsMap>({
                     (dimensionFilterRule) => {
                         const isNotSelectedFilter =
                             dimensionFilterRule.id !== filterId;
-                        const hasSameTable =
-                            dimensionFilterRule.target.tableName === item.table;
-                        return isNotSelectedFilter && hasSameTable;
+                        return isNotSelectedFilter;
```

Comment:
Sends all dashboard filters in API requests and let the BE find which ones it can use.